public/app/features/alerting/unified/Templates.test.tsx

```diff
@@ -38,6 +39,15 @@ jest.mock('@grafana/ui', () => ({
   ),
 }));
 
+const successMock = jest.fn();
+jest.mock('app/core/copy/appNotification', () => {
+  return {
+    useAppNotification: () => {
+      return { success: successMock };
+    },
+  };
+});
```

Comment:
(Going to add a commit to remove the mock, as we can still render `AppNotificationList` in the setup method and assert it that way :+1:)